kc-cloudtrust-module/kc-cloudtrust-common/src/main/java/io/cloudtrust/keycloak/ThemeUtils.java

```diff
@@ -19,20 +19,34 @@
     /**
      * Look for the theme of the current realm
      *
+     * @param session The Keycloak session
      * @param type Theme type
      * @return A type (if any was found)
      * @throws IOException
      */
     public static Theme findTheme(KeycloakSession session, Theme.Type type) throws IOException {
         RealmModel realm = session.getContext().getRealm();
         String loginTheme = realm.getLoginTheme();
```

Comment:
There an issue here... If type==Type.LOGIN, that's ok... but for other values, we should not getLoginTheme() 🤔

Reply:
Okay, I fixed it ;-)